greenheart/tools/eco/finance.py

```diff
@@ -1461,6 +1465,21 @@ def run_profast_full_plant_model(
         cost=opex_breakdown["electrolyzer"],
         escalation=gen_inflation,
     )
+    if isinstance(electrolyzer_cost_info["electrolyzer_var_om"], (list, np.ndarray)):
```

Comment:
same problem here as with the capex, we need to make sure the costs are discounted, preferably in the same place and way the other costs are discounted.